sansio_lsp_client/events.py

```diff
@@ -78,3 +130,98 @@ class WillSaveWaitUntilEdits(Event):
 class PublishDiagnostics(ServerNotification):
     uri: str
     diagnostics: t.List[Diagnostic]
+
+
+class Hover(Event):
```

Comment:
What do the comments in this class body mean?

Reply:
They listed attributes that the members had, which is just duplicate information, because you can go to where the member type is defined and look. I deleted them.